src/lib/shiori-loader.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * 'fs' - node.js/BrowserFS のファイルシステム
+ */
+export type FileSystemLike = any;
+
+/**
+ * SHIORIサブシステム
+ */
+export interface Shiori {
```

Comment:
__[tslint]__
* interface name must be a capitalized I

<!-- Comment by [SideCI](https://sideci.com) -->